contrib/test.sh

```diff
@@ -41,6 +42,9 @@ if [ "$DO_FEATURE_MATRIX" = true ]; then
 		done
     done
 
+    # Other combos
+    # TODO: Add this test once we bump MSRV past 1.29
+    # cargo test --all --no-default-features --features="std,schemars"
```

Comment:
In af992d7e9abd4acd0fa0ec1b622e35498d1f6971:

Did you intend to comment out this line? From your commit message it sounds like you *do* intend to test std and schemars.

We definitely need schemars to be tested in CI somehow. Historically we have had breakage from this dependency that went undetected by CI. (This is why, incidentally, I worte all my own independent CI scripts, and also why schemars is not a dependency in any other rust-bitcoin crate.)

Reply:
Shit, I've been a bit sloppy lately fixing commit messages when re-basing. I'm still struggling big time with Rust 1.29, I have some shell functions to build with 1.29 but I often forget to use them because they are not well tied in with my editor ... so I'm constantly re-basing and getting punished by CI. You might have noticed all the force pushes in my PRs :) I'm being lazy, we are so close to bumping to 1.41

We have `extended_tests` which test `bitcoin_hashes` with `serde`, `schemars`, and `std` enabled. The `schemars` feature doesn't build on its own at the moment `cargo check --no-default-features --features=schemars` fails. I didn't investigate why I just added this comment to remind us.

I added this commented out line with the MSRV comment since I've seen that done in a few places and have started doing it. Now as I write this message I think it would be better to add all these to the edition 2018 tracking issue instead of leaving it in the code. 

Will fix the commit log.

Reply:
Ok, good observation that we'll be bumping to 1.41.0 soon, and at that point we can put schemars into the normal CI pipeline.